UI/appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "ConnectionStrings": {
```

Comment:
Just FYI: such values as connection string or similar, which could contain sensitive data, it's better to keep in secret.json file. Read about it. (Hint -> in Solution explorer -> mouse right button click on UI project -> Manage User Secrets)

Reply:
will do

Reply:
added secret